### ISSUE TITLE
Feature: Rebase off of Sway 1.10.1

### ISSUE THREAD

**Issue by markstos** [WillPower3309/swayfx#360]
Sway 1.10 has been released. It would be great if `swayfx` could be rebased to be based on  the latest version of Sway and wlroots.

**Comment by ErikReider** [comment]
Requires https://github.com/wlrfx/scenefx/pull/55

**Comment by WillPower3309** [comment]
Also https://github.com/wlrfx/scenefx/pull/69 and https://github.com/wlrfx/scenefx/pull/33

**Comment by markstos** [comment]
I just tried upgrading from Sway 1.9 to 1.10 on a machine and it was a hot mess. Fuzzel would no longer launch, my `wmfocus` window switching utility became unreliable and sometimes I couldn't click into terminals and type.

That is to say, I wouldn't worry about rushing this. There are alread Sway several issues tagged with 1.10.1, and I wouldn't recommend targeting any earlier release of the 1.10 series than 1.10.1.

**Comment by WillPower3309** [comment]
> I just tried upgrading from Sway 1.9 to 1.10 on a machine and it was a hot mess. Fuzzel would no longer launch, my `wmfocus` window switching utility became unreliable and sometimes I couldn't click into terminals and type.
> 
> That is to say, I wouldn't worry about rushing this. There are alread Sway several issues tagged with 1.10.1, and I wouldn't recommend targeting any earlier release of the 1.10 series than 1.10.1. 

Agreed! We discussed this a bit on discord, will wait for 1.10.1 to avoid all the mess

**Comment by markstos** [comment]
All the blockers for this have been resolved. The three PRS it depended on have been merged and Sway 1.10.1 was released.

**Comment by WillPower3309** [comment]
> All the blockers for this have been resolved. The three PRS it depended on have been merged and Sway 1.10.1 was released.

Need to merge https://github.com/WillPower3309/swayfx/pull/367 first. Then 1.10.1 will be a small + fast follow-up

**Comment by WillPower3309** [comment]
We decided to do a 1.10.0 merge first since it effectively required rewriting swayfx
